.github/workflows/kotlin_localstack_validate.yml

```
@@ -0,0 +1,78 @@
+name: validate template for kotlin with localstack and postgres
+on:
+  workflow_call:
+    inputs:
+      working_directory:
+        type: string
+        required: false
+        default: '~/app'
+      java_version:
+        type: string
+        required: false
+        default: '21'
+      java_options:
+        type: string
+        default: '-Xmx512m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=1'
```

```suggestion
        default: '-Xmx768m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=1'
```
circleci had bigger default settings

---

although saying that the build succeeded 🤷